R/occurrenceChange.r

```diff
@@ -10,6 +10,23 @@
 #' @param firstYear numeric, the first year over which the change is to be estimated
 #' @param lastYear numeric, the last year over which the change is to be estimated
 #' @param bayesOut occDet object as returned from occDetModel
+#' @param change A character string that specifies the type of change to be calculated, the default
```

Comment:
You might want to add 'see details' so that people know to look there for the values it can take